### ISSUE TITLE
[$20][MSFT-230] Login page: Colour contrast issues.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2888]
**Steps to Reproduce**
1. Go to the Members login:https://accounts.topcoder.com/member

2. Please run the automated tool AXE. 3. Capture the colour contrast issues.

**Expected Result**
The colour contrast should comply as per the WCAG minimum contrast requirements.

**Actual Result**
The colour contrast between the background and foreground of the elements doesn't comply with WCAG minimum contrast requirements.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.1.1 Non-Text Content
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2040

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096239 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096239 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by r0hit-gupta** [comment]
Please assign me

**Comment by r0hit-gupta** [comment]
PR at https://github.com/appirio-tech/accounts-app/pull/236

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096239 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by veshu** [comment]
@r0hit-gupta I hope you are passing minimum AA WCAG level of contrast ratio

**Comment by r0hit-gupta** [comment]
@veshu yes I am passing the minimum AA WCAG level of contrast ratio

**Comment by veshu** [comment]
Thanks now you should be able to assign and update label. Could you please check and let me know

**Comment by r0hit-gupta** [comment]
@veshu yes it works now. Thanks a lot.

**Comment by Dara-K** [comment]
You can use these color updates:
Text link "Forgot your password": #006DEA
Github, Gooogle, Facebook, etc. + "Or LOGIN WITH":  #6B6B6B
Show Button: let’s have a white button, border color #C3C3C8 and text color #47474F
Copyright text: #6B6B6B

**Comment by r0hit-gupta** [comment]
@Dara-K I have changed the colors. Thanks.

**Comment by tosha5252** [comment]
Forgot your password: set at #0079ce, instead of #006DEA
 
Env: 
Windows 10
Browser: Chrome, up to date

**Comment by nithyaasworld** [comment]
@veshu @tosha5252 

1. "Forgot your password" and "Join Now" links still have the old colour.
2. The colours are hard-coded instead of changing the variable's value. Please confirm if this is a good practice. For a similar issue, the other developer has fixed the variable [here](https://github.com/appirio-tech/accounts-app/pull/222/files) instead of hard-coding which looks good to me.

Thanks.

**Comment by nithyaasworld** [comment]
CC: @Dara-K @mishacucicea @nkumar-topcoder

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2888 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2888 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097088 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097088 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by r0hit-gupta** [comment]
@veshu do I create a new PR for this?

**Comment by veshu** [comment]
@r0hit-gupta Yes please create new PR, old must have been already reverted.

cc @mishacucicea

**Comment by r0hit-gupta** [comment]
@veshu accounts-app uses a different color palette https://github.com/appirio-tech/accounts-app/blob/dev/app/styles/tc-ui-variables.scss

Do I simply change the value of some variables or update the whole color palette according to https://marvelapp.com/c7h2a6h/screen/59648056/handoff ?

**Comment by veshu** [comment]
@Dara-K Please guide here, variable names look different than react ui kit.

@mishacucicea Please assist here. Thanks!

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097088 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by mishacucicea** [comment]
@r0hit-gupta I would say to update only the variable values that are impacting this. Raising a bit the prize on this one, please make sure you take your time when verifying the change before the PR, so that we don't break anything.
@nkumar-topcoder Will `https://github.com/appirio-tech/accounts-app/blob/dev/app/styles/tc-ui-variables.scss`? be replaced with something else in the future?

**Comment by r0hit-gupta** [comment]
@mishacucicea please review.

**Comment by mishacucicea** [comment]
@r0hit-gupta Contrast is not good :( 
![image](https://user-images.githubusercontent.com/1721199/61999810-0715ca00-b0d2-11e9-86b1-9f5d2c9bc845.png)

**Comment by r0hit-gupta** [comment]
@mishacucicea fixed. I am sorry, the issue did not mention the login button so skipped it earlier. Please review now :)

**Comment by tosha5252** [comment]
Verified changed to #006AD7

OS: Windows 10
Browser: Chrome

**Comment by nkumar-topcoder** [comment]
@tosha5252 actually its not #006AD7, the PR merge file has #0096ff; can you recheck

**Comment by tosha5252** [comment]
@nkumar-topcoder yes I will re-check

**Comment by tosha5252** [comment]
@nkumar-topcoder Indeed there were changes to the new color 006ad7...but sometimes it reverts back for some reason...I think some people push more than the branch that is changed sometimes? or either there are dup tickets...I see that some things have gone back...but there are 0 contrast issues at this time....which we were told to move on as passing until after the end of this run...

![Capture-1](https://user-images.githubusercontent.com/6400904/62105898-4c203480-b269-11e9-8be3-047db30ce42b.PNG)

**Comment by lakshmiathreya** [comment]
Verified Log In is fixed to #006ad7.  @nkumar-topcoder - could you let us know what the issue is since this color seems correctly set as per Designer recommendation?

**Comment by nkumar-topcoder** [comment]
ok fine will move ahead with Dev merge @lakshmiathreya

**Comment by tosha5252** [comment]
Passed as per comments

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097088<br/><br/>```This is an automated message for crazyk via Topcoder X```
